src/Compilers/CSharp/Test/Symbol/BadSymbolReference.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
@gafter I can confirm, it looks like my script to change out the [Fact] attributes saved the files without the BOM.